src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,31 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import duck.Task;
+import duck.TaskList;
+
+public class TaskListTest {
+
+    @Test
+    public void addTaskTest() {
+        ArrayList<Task> list = new ArrayList<Task>();
+        TaskList taskList = new TaskList(list);
+        Task task = new Task(false, "sleep");
+        taskList.add(task);
+        assertEquals(1, taskList.size());
+        assertEquals(task, taskList.get(0));
+    }
+    @Test
+    public void deleteTaskTest() {
```

Comment:
Should methods be separated by line break?

```suggestion
        assertEquals(task, taskList.get(0));
    }

    @Test
    public void deleteTaskTest() {
```